src/main/java/com/google/idea/perf/TracerConfig.kt

```diff
@@ -43,6 +43,13 @@ object TracerConfig {
         lock.withLock {
             val classConfig = classConfigs.getOrPut(classJvmName) { ClassConfig() }
             classConfig.methodNames.add(methodName)
+
+            for ((signature, methodId) in classConfig.methodIds) {
+                if (signature.startsWith(methodName)) {
```

Comment:
Using `startsWith` here could be inaccurate in some cases. I think we could use `signature.substringBefore('(')` to get the method name from the signature.